tools/generators/workspace-generator/files/lib/utils.ts__tmpl__

```diff
@@ -0,0 +1,5 @@
+// use this module to define any kind of generic utilities that are used in more than 1 palce within the generator implementation
```

Comment:
I created that to explicitly demonstrate the scaffolding that is mentioned in docs. people tend to not read docs at all because they usually became obsolete very quickly.  

To me this doesn't add any value and I agree what you said. would like to hear others opinion.